templates/etc/profile.d/tlog.sh.epp

```diff
@@ -0,0 +1,62 @@
+<%- |
+  Stdlib::Absolutepath $users_file = '/etc/security/tlog.users',
+  Stdlib::Absolutepath $app_path   = '/usr/bin/tlog-rec-session'
+| -%>
+# Place this script in /etc/profile.d to automatically hook any login or
+# interactive shell into tlog for a user or group listed in
+# /etc/security/tlog.users
+#
+# Entries in tlog.users should be listed one per line where users are bare
+# words such as `root` and groups are prefixed with a percent sign such as
+# `%root`.
+#
+# Copyright 2018 Trevor Vaughan <tvaughan@onyxpoint.com> - Onyx Point, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+TLOG_USERS="<%= $users_file %>"
+TLOG_CMD="<%= $app_path %>"
+
+tlog_parent(){
+  retval=1
+
+  ppid=`ps --no-headers -o ppid $1`
+
+  if [ $ppid -gt 1 ]; then
```

Comment:
I get an error that  -bash: [: -gt: unary operator expected  when logging in.  Need [[   $ppid -gt 1 ]] I believe.

Reply:
The issue here was that `$ppid` was not actually an item. Working on a patch around this.